### ISSUE TITLE
Issues with using pre-trained weights Resnet50.npy with RetinaNet

### ISSUE THREAD

**Issue by dunyasik** [xmyqsh/RetinaNet#2]
Hi, I have tried several pre-trained Resnet50.npy models to train the RetinaNet using the exact commands provided in the readme here.
However, after 40 50 iterations the loss becomes NaN everytime.

Example pre-trained model I tried:
https://lexiondebug.blob.core.windows.net/mlmodel/models/res50.npy

Could you provide the pre-train model you use to train your models, or please guide me what else could be the problem here?
Thanks alot

**Comment by xmyqsh** [comment]
https://drive.google.com/file/d/0B_xFdh9onPagSWU1ZTAxUTZkZTQ/view

**Comment by dunyasik** [comment]
Thanks a lot!!! seems to be working fine.
will provide further feedback when training finishes.
